realm/realm-library/src/main/java/io/realm/internal/ObjectServerFacade.java

```diff
@@ -66,7 +65,7 @@ public void realmClosed(RealmConfiguration configuration) {
     }
 
     public Object[] getSyncConfigurationOptions(RealmConfiguration config) {
-        return new Object[12];
+        return new Object[14];
```

Comment:
Is this change intended? Also, avoid using magic numbers.

Reply:
Yes, good point. Added a constant with this that is shared.